test/cppsuite/test_harness/runtime_monitor.cxx

```diff
@@ -236,6 +226,16 @@ postrun_statistic_check::check_stat(scoped_cursor &cursor, const postrun_statist
 }
 
 /* runtime_monitor class implementation */
+void
+runtime_monitor::get_stat(scoped_cursor &cursor, int stat_field, int64_t *valuep)
```

Comment:
Is it thread-safe?

Reply:
This function is not thread safe, if multiple threads uses the same stat cursor to call `get_stat()`. Do you think we should open the cursor within the function instead?

Reply:
Yes, I think it would be better, we might need to rethink how the `_cursor` member is used in this class.